mobile_index/src/main.rs

```diff
@@ -0,0 +1,76 @@
+use clap::Parser;
+use mobile_index::{settings::Settings, Indexer, Result};
+use std::path::PathBuf;
+use tokio::signal;
+use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
+
+#[derive(Debug, clap::Parser)]
+#[clap(version = env!("CARGO_PKG_VERSION"))]
+#[clap(about = "Helium Mobile Indexer")]
```

Comment:
This value should appear in Cargo.toml for one place to maintain.
```suggestion
#[clap(about = env!("CARGO_PKG_DESCRIPTION"))]
```
See https://doc.rust-lang.org/cargo/reference/environment-variables.html